tests/notebooks/annotation.ipynb

```diff
@@ -66,6 +60,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "assert n_samples_tsne == 10\n",
+    "assert n_epochs_all == 1\n",
+    "assert n_labelled_samples_per_class == 3\n",
+    "assert M_sampling_all == 1\n",
```

Comment:
Does this notebook run when it's not run by unit tests? Wouldn't these assert statements be false?

Reply:
Thanks for pointing this out, I forgot to delete it after testing that the parameters's values were replaced, I'll commit this fix